tensorflow/python/eager/function.cc

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <Python.h>
+
+#include "pybind11/pybind11.h"
+#include "pybind11/stl_bind.h"
+
+struct ConcreteFunction;  // Forward declaration.
+
+// TODO(jlchu): Migrate Python characteristics to C++
+
+namespace tensorflow {
+
+namespace py = pybind11;
+
+py::module* np_arrays = new py::module(
+    py::module::import("tensorflow.python.ops.numpy_ops.np_arrays"));
+py::module* nest = new py::module(
+    py::module::import("tensorflow.python.util.nest"));
```

Comment:
If it doesn't work, make it `static const` and move `inside _build_call_outputs()` function.  Like how `tf_stack.cc` imports "linecache".

Reply:
Done